### PR TITLE
fix apy and add new redemption contract

### PR DESCRIPTION
**Pull request by mel0ndev**

### CODE REVIEW

File: src/store/vaults.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ const calculateApyFromRewardRate = (
     const _numTokensPurchasedByReward = _usdRewardsPerYear / tokenPrice;
     const _aprDecimal = _numTokensPurchasedByReward / _totalStaked;
     if (isNaN(_aprDecimal)) return 0;
-    return _aprDecimal;
+    return _aprDecimal / 100; //stop gap fix, not sure where everything being scaled, TODO fix this later
```
Comment (clothic) [inline src/store/vaults.tsx:61]: Why is this changed

Comment (mel0ndev) [reply src/store/vaults.tsx:61]: aprs are scaled improperly without it, went from 200% to 2k%

**Review by yoyobojo — APPROVED** [review]
